src/one_dragon/base/operation/operation_notify.py

```
@@ -254,9 +286,15 @@ def send_node_notify(
-    # 异步推送
-    operation.ctx.push_service.push_async(
-        title=operation.ctx.notify_config.title,
-        content=message,
-        image=operation.last_screenshot if send_image else None,
-    )
+    image = operation.last_screenshot if send_image else None
+
+    # 收集到通知池
+    pool.add(content=message, image=image)
+
+    # ALL 等级时逐条发送；MERGE 等级时仅收集，但失败时也立即发送
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)